assets/src/edit-story/elements/text/edit.js

```diff
@@ -205,13 +196,29 @@ function TextEdit({
   // Update content for element on unmount.
   useUnmount(updateContent);
 
+  useEffect(() => {
+    // If there are any moveable actions happening, let's update the content
+    // Otherwise the font size and measures will not be correct.
+    if (actionHappening) {
+      updateContent();
+    }
+  }, [actionHappening, updateContent]);
+
   // A function to remeasure height
   const handleResize = useCallback(() => {
     const wrapper = wrapperRef.current;
     const textBox = textBoxRef.current;
     editorHeightRef.current = textBox.offsetHeight;
     wrapper.style.height = `${editorHeightRef.current}px`;
-  }, []);
+    if (editWrapper) {
+      // @todo Get the correct top and left from getBox, just height for testing currently!
+      editWrapper.style.height = `${editorHeightRef.current}px`;
+      // We need to update Moveable as well.
+      if (moveable.current) {
```

Comment:
The height has been updated (the element properties not yet), we need to visually update Moveable as well to reflect that.

Reply:
Maybe it would be better as a callback, e.g. `onResize`? And then handle the updating in `EditElement`?

Reply:
Hmm, here the updating of the element is specifically related to Text edit mode only -- changing text is causing the resize. It seems to make more sense to have it here since it seems to be very specific to Text edit, even if theoretically in the future another element would have moveable + edit mode together. Thoughts?

Reply:
It's mostly the fact, that the `TextEdit` component now for weird reasons (as a component) expects an argument, which is a ref to something with an `updateRect` method. That's a rather weirdly specific requirement for this component, that doesn't really make sense if this component is used anywhere else (not that I can think of where). For the sake of SoC, it makes more sense to simply pass a `handleResize` and invoke it inside any editable component (images and videos might be able to resize as well).

Reply:
Makes sense, moved the moveable part to `onResize`, PTAL: 15f9a26

Moving to QA meanwhile.